### ISSUE TITLE
TinyMCE Angular Events not firing on initialization 

### ISSUE THREAD

**Issue by treywilson** [tinymce/tinymce-angular#151]
**What is the current behavior?**
- If I use the `onBeforeSetContent` and `onSetContent` events within my Angular `editor` component, the value I provide on initialization (`(ngModel)="myValue"`) does not fire the two events.

If I instead set the events up within the `initObject`:
```
    setup: editor => {
      editor.on('BeforeSetContent', e => this.beforeSetContentEvent(e));
      editor.on('SetContent', e => this.setContentEvent());
    }
``` 
The events are correctly fired on initialization.

**Please provide the steps to reproduce and if possible a minimal demo of the problem via [codesandbox.io](https://codesandbox.io/s/6zlmmkkk1n?fontsize=14) or similar.**
- https://codesandbox.io/s/tinymceangular-ik8es
- Check out the logs in the console. You will see that the init object correctly logs out, but the angular events do not log out on load. Press the Set Content button to see both fire correctly.

**What is the expected behavior?**
- For the events used in the markup to fire on initialization just like the ones setup through the `init` are
**Which versions of TinyMCE/TinyMCE-Angular, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE or TinyMCE-Angular?**
- My current version is `3.2.0` but the one you guys linked for the sandbox looks like it is `3.3.0` so it affects that version too. Not sure if this ever worked in previous versions.

**Comment by jscasca** [comment]
Yes, that makes perfect sense. We were binding the angular events *after* doing the initial set content but the editor init already ran setting the other editor events. I will make a PR to fix this

**Comment by treywilson** [comment]
Awesome. Thanks @jscasc!

**Comment by jscasca** [comment]
Sorry, I forgot to update this issue. This was addressed in 3.5.2 on May 13 (https://github.com/tinymce/tinymce-angular/pull/162)
